.github/workflows/expectfail.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 if $1; then
-    echo "FALSE POSITIVE! Expected failure but succeeded"
-    exit 1
```

Comment:
This file is not `expectfail.sh` any longer... it's expecting it to pass.
You could remove this file and directly run the executable. It will have the same effect